### PR TITLE
custom landing page: include custom head scripts

### PR DESCRIPTION
**Pull request by valdis**

### CODE REVIEW

**Comment by valdis** [comment]
test failures are unrelated to implementation

File: app/controllers/homepage_controller.rb [hunk 1]
```diff
@@ -139,6 +140,13 @@ def index
   # rubocop:enable AbcSize
   # rubocop:enable MethodLength
 
+  include ActionView::Helpers::JavaScriptHelper
+
+  def custom_head_scripts
```
Comment (sktoiva) [inline app/controllers/homepage_controller.rb]: I guess these changes should be in landing_page_controller?

Comment (thomasmalbaux) [reply app/controllers/homepage_controller.rb]: @sktoiva @valdis Is this still a relevant change request?

Comment (valdis) [reply app/controllers/homepage_controller.rb]: @sktoiva @thomasmalbaux issues resolved

**Pull request merged**